<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ModelVerificationWindow</class>
 <widget class="QDialog" name="ModelVerificationWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>673</width>
    <height>431</height>
   </rect>
  </property>
  <property name="mouseTracking">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>Model Verification Window</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>631</width>
     <height>121</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label_model">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
          <weight>50</weight>
          <italic>false</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>Choose model:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboBox_model"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="label">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
          <weight>50</weight>
          <italic>false</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>Choose UBM:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboBox_ubm"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="label_threshold">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Choose threshhold</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="doubleSpinBox_threshold">
        <property name="minimum">
         <double>-100.000000000000000</double>
        </property>
        <property name="maximum">
         <double>100.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.100000000000000</double>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>631</width>
     <height>231</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab_fromFilesystem">
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <attribute name="title">
     <string>From Filesystem</string>
    </attribute>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>150</y>
       <width>611</width>
       <height>41</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="QPushButton" name="pushButton_add_rec_filesystem_2">
        <property name="text">
         <string>Add Record</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_remove_filesystem_2">
        <property name="text">
         <string>Remove Record</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QListWidget" name="listWidget_records_filesystem_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>611</width>
       <height>111</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>45000</width>
       <height>45000</height>
      </size>
     </property>
     <property name="baseSize">
      <size>
       <width>200</width>
       <height>200</height>
      </size>
     </property>
     <property name="mouseTracking">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_filesystem_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>611</width>
       <height>30</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>20</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>30</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Choose records to verifiing</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_fromMicrophone">
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <attribute name="title">
     <string>From Microphone</string>
    </attribute>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>150</y>
       <width>611</width>
       <height>41</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_8">
      <item>
       <widget class="QPushButton" name="pushButton_add_rec_microphone">
        <property name="text">
         <string>Add Record</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_remove_microphone">
        <property name="text">
         <string>Remove Record</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QListWidget" name="listWidget_records_microphone">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>39</y>
       <width>611</width>
       <height>111</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>4999</width>
       <height>5000</height>
      </size>
     </property>
     <property name="baseSize">
      <size>
       <width>200</width>
       <height>200</height>
      </size>
     </property>
     <property name="mouseTracking">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_filesystem_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>611</width>
       <height>30</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>20</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>30</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Choose records to verifiing</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>380</y>
     <width>311</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QPushButton" name="pushButton_cancell">
      <property name="text">
       <string>Cancell</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_verify">
      <property name="text">
       <string>Verify</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
